archiver/direct.py

```diff
@@ -86,23 +84,24 @@ def direct_archiver_from_params(
 
     biosamples_client = BioSamples(aap_client, biosamples_url)
     biosamples_converter = BioSamplesConverter(biosamples_domain)
-    biosamples_submitter_service = BioSamplesSubmitterService(biosamples_client, deployment_env)
+    biosamples_submitter_service = BioSamplesSubmitterService(biosamples_client)
     biosamples_submitter = BioSamplesSubmitter(biosamples_client, biosamples_converter, biosamples_submitter_service)
 
-    if deployment_env.lower() not in ('dev', 'staging'):
-        # TODO when we solved the issue with BioStudies availability, then we can remove the above condition
+    # TODO when we solved the issue with BioStudies availability, then we can remove the above condition
```

Comment:
is there a ticket for it? would be nice to track that work/conversation on the ticket.